addons/web/static/src/views/list/list_controller.js

```diff
@@ -332,7 +332,7 @@ export class ListController extends Component {
                 callback: () => this.toggleArchiveState(false),
             },
             duplicate: {
-                isAvailable: () => this.activeActions.duplicate && !isM2MGrouped,
+                isAvailable: () => this.activeActions.duplicate && this.activeActions.create && !isM2MGrouped,
```

Comment:
It's weird to say that we can duplicate (`this.activeActions.duplicate`) if we can't create. So I would rather fix this in views/utils.js (getActiveActions), as follows:
```js
    const activeActions = {
        type: "view",
        edit: archParseBoolean(rootNode.getAttribute("edit"), true),
        create: archParseBoolean(rootNode.getAttribute("create"), true),
        delete: archParseBoolean(rootNode.getAttribute("delete"), true),
    };
    activeActions.duplicate =
        activeActions.delete && archParseBoolean(rootNode.getAttribute("duplicate"), true);
    return activeActions;
```

Reply:
@aab-odoo ,

requested changes done.